src/EventSubscriber/CspEventSubscriber.php

```diff
@@ -95,16 +212,9 @@ public function policyAlter(PolicyAlterEvent $event): void {
         $this->environmentResolver->getActiveEnvironmentName()
```

Comment:
Does this work with avustusasiointi, which has `dev` environment? (etusivu does not have dev environment).

getActiveEnvironment [seems to throw](https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/blob/main/src/Environment/Project.php#L150) if the project does not have the requested environment.

Reply:
Wow, good catch! Added a try…catch to handle that exception gracefully.